www/src/templates/template-docs-markdown.js

```diff
@@ -84,6 +87,28 @@ class DocsTemplate extends React.Component {
                 {page.frontmatter.title}
               </h1>
               <div
+                css={{
+                  "> .gatsby-code-title": {
+                    backgroundColor: colors.gatsby,
+                    borderTopLeftRadius: presets.radiusLg,
+                    borderTopRightRadius: presets.radiusLg,
+                    color: `white`,
+                    fontFamily: options.monospaceFontFamily.join(`,`),
+                    ...scale(-1 / 5),
+                    marginLeft: rhythm(-options.blockMarginBottom),
+                    marginRight: rhythm(-options.blockMarginBottom),
+                    padding: `${rhythm(options.blockMarginBottom / 2)} ${rhythm(
+                      options.blockMarginBottom
+                    )}`,
+                    [MIN_DEFAULT_MEDIA_QUERY]: {
+                      marginLeft: rhythm(-options.blockMarginBottom * 1.5),
+                      marginRight: rhythm(-options.blockMarginBottom * 1.5),
+                      padding: `${rhythm(options.blockMarginBottom)} ${rhythm(
+                        options.blockMarginBottom * 1.5
+                      )}`,
+                    },
+                  },
+                }}
```

Comment:
Could this be abstracted so codeblock `title` could be used in blog posts as well? Not high priority, but would be nice if it's easy to do.

Reply:
Good idea. I should probably just add it in the src/util/typography file instead of at the component level! I'll do that tonight probably.